admin_guide/topics/testing-auto-scaler.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='testing-AWS-cluster-auto-scaler_{context}']
+= Testing the auto-scaler
+
+After you install or upgrade your cluster that uses the auto-scaler, you can
+confirm that auto scaling works.
+
+.Prerequisites
+
+* Your {product-title} cluster on Amazon Web Services (AWS) uses the
+auto-scaler.
+
+
+. Create the *_scale-up.yaml_* file that contains the deployment configuration
+to test auto-scaling:
++
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: scale-up
+  labels:
+    app: scale-up
+spec:
+  replicas: 20
+  selector:
+    matchLabels:
+      app: scale-up
+  template:
+    metadata:
+      labels:
+        app: scale-up
+    spec:
+      containers:
+      - name: busybox
+        image: docker.io/library/busybox
+        resources:
+          requests:
+            memory: 2Gi
+        command:
+        - /bin/sh
+        - "-c"
+        - "echo 'this should be in the logs' && sleep 86400"
+      terminationGracePeriodSeconds: 0
+----
++
+This deployment specifies 20 replicas, but the initial size of the cluster
+cannot run all of the pods without first increasing the number of compute nodes.
+
+. Deploy the configuration:
++
+[source,bash]
+----
+$ oc apply -n autoscaler-demo -f scale-up.yaml
```

Comment:
this references the demo namespace from the other doc, where i think you had wanted the openshift-* namespace instead.

Reply:
@derekwaynecarr address all your comments. PTAL.

Reply:
For this I had intended it to be the demo namespace. Deployment should work outside of the cluster-autoscaler namespace -- essentially where the CA is now deployed.